script.js

```diff
@@ -17,46 +17,49 @@ const themeMode = () => {
     }
 }
 
+let _data
 
 // Fetching Topics
-let topics = [];
-let filteredTopics = []
+
 const fetchTopics = async (searchQuery) => {
     try {
         const response = await fetch(searchQuery
             ? `https://tap-web-1.herokuapp.com/topics/list?phrase=${searchQuery}`
             : `https://tap-web-1.herokuapp.com/topics/list`);
-        topics = await response.json();
-        filteredTopics = [...topics];
+        let topics = await response.json();
+       let filteredTopics = [...topics];
```

Comment:
why you are cloning the topics here? it has no meaning, since the original object `topics` is not even used or saved. after this function ends, the original object will be destroyed.